scripts/driver/driver_internal.sh

```diff
@@ -88,15 +88,16 @@ kubectl create namespace "$NAMESPACE"
 
 function delete_namespace() {
   echo "INFO Collecting events for namespace \"$NAMESPACE\""
-  kubectl get events --namespace=$NAMESPACE || echo "ERROR Failed to get events for namespace $NAMESPACE"
+  kubectl get events --namespace="$NAMESPACE" || echo "ERROR Failed to get events for namespace \"$NAMESPACE\""
 
   if [[ ! -z $deployer_name ]]; then
     echo "INFO Collecting logs for deployer"
     kubectl logs "jobs/$deployer_name" --namespace="$NAMESPACE" || echo "ERROR Failed to get logs for deployer $deployer_name"
   fi
 
   echo "INFO Deleting namespace \"$NAMESPACE\""
-  kubectl delete namespace $NAMESPACE
+  kubectl get all --namespace "$NAMESPACE" --output=yaml
```

Comment:
just a note: `get all` doesn't cover all resources, specifically RBAC resources and PVCs